src/screens/TopicMemberScreen/index.js

```diff
@@ -206,6 +158,7 @@ const TopicMemberScreen = () => {
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
+        domain={topicName}
         animatedHeight={animatedHeight}
         onShareCommunity={onCommunityShare}
         isHeaderHide={headerHide}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `domain` prop has been added to the `NavHeader` component. Make sure that this new prop is properly handled in the `NavHeader` component.
```diff
+ domain={topicName}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->